app/databox/databox.py

```diff
@@ -78,13 +78,31 @@ def is_builtin(command_id):
     'large_sigma': Command('touch', '/tmp/created-by-large_sigma'),
 }
 
+
+def is_active(symbol):
+    """Check if given symbol is active.
+
+    if there is no such symbol in database, returns false"""
```

Comment:
- [ ] Args are missing.
- [ ] This function returns `None` sometimes so _if there is no such symbol in database, returns false_ is not true.